app/models/release.rb

```diff
@@ -44,6 +44,9 @@ def self.find_by_version!(version)
   private
 
   def assign_release_number
+    default_number = 1
```

Comment:
the reason you added this code is to be able to overwrite the release with a number ... that is not super obvious from the code ... and can be removed without breaking any tests ... so some docs + a test would be nice